include/mbedtls/ctr_drbg.h

```diff
@@ -210,6 +210,11 @@ mbedtls_ctr_drbg_context;
  *                      and prepares it for mbedtls_ctr_drbg_seed()
  *                      or mbedtls_ctr_drbg_free().
  *
+ * \note                The reseed interval is
+ *                      #MBEDTLS_CTR_DRBG_RESEED_INTERVAL by default.
+ *                      You can override it by calling
+ *                      mbedtls_ctr_drbg_set_reseed_interval().
+ *
```

Comment:
Since this is a bug, we should fix it in the long-time support branches as well (`mbedtls-2.16` and `mbedtls-2.7`). Would you mind submitting a similar patch (including the changelog entry) to those two branches?